### PR TITLE
Add user interface

### PR DESCRIPTION
**Pull request by kene-creator**
- Initialized React app
-  Created reusable components
- Displayed list of books
- Created a component called Book for displaying a single book
- Displayed a form for adding a book 
- Created a separate component for this form
- Category page displays "check status"

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,65 @@
+/* eslint-disable no-unused-vars */
```
Comment (cynthiainga) [inline src/App.js]: - [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉

File: src/components/UI/BookList.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+
```
Comment (cynthiainga) [inline src/components/UI/BookList.js]: - [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉

File: src/components/UI/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-unused-vars */
```
Comment (cynthiainga) [inline src/components/UI/Navbar.js]: - [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉

File: src/index.js [hunk 4]
```diff
@@ -1,13 +1,19 @@
+/* eslint-disable no-unused-vars */
 /* eslint comma-dangle: ["error", "never"] */
+
 import React from 'react';
 // eslint-disable-next-line
```
Comment (cynthiainga) [inline src/index.js]: - [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉

File: src/pages/Categories.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
```
Comment (cynthiainga) [inline src/pages/Categories.js]: - [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean codebase 😁 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @kene-creator, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cwaku — APPROVED** [review]
Hi @kene-creator 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**